src/components/YieldPools/ElasticFarmGroup/FarmCard.tsx

```diff
@@ -0,0 +1,479 @@
+import { Currency, CurrencyAmount } from '@kyberswap/ks-sdk-core'
+import { Trans } from '@lingui/macro'
+import dayjs from 'dayjs'
+import { darken } from 'polished'
+import { useState } from 'react'
+import { Minus, Plus, Share2 } from 'react-feather'
+import { Link } from 'react-router-dom'
+import { Flex, Text } from 'rebass'
+import styled from 'styled-components'
+
+import bgimg from 'assets/images/farm-card-background.png'
+import { ReactComponent as DropIcon } from 'assets/svg/drop.svg'
+import { ButtonEmpty, ButtonLight } from 'components/Button'
+import CopyHelper from 'components/Copy'
+import CurrencyLogo from 'components/CurrencyLogo'
+import Divider from 'components/Divider'
+import DoubleCurrencyLogo from 'components/DoubleLogo'
+import HoverDropdown from 'components/HoverDropdown'
+import { Swap as SwapIcon } from 'components/Icons'
+import Harvest from 'components/Icons/Harvest'
+import InfoHelper from 'components/InfoHelper'
+import { MouseoverTooltip } from 'components/Tooltip'
+import { APP_PATHS, ELASTIC_BASE_FEE_UNIT } from 'constants/index'
+import { TOBE_EXTENDED_FARMING_POOLS } from 'constants/v2'
+import { useActiveWeb3React } from 'hooks'
+import useTheme from 'hooks/useTheme'
+import { FarmingPool, NFTPosition } from 'state/farms/elastic/types'
+import { shortenAddress } from 'utils'
+import { getFormattedTimeFromSecond } from 'utils/formatTime'
+import { formatDollarAmount } from 'utils/numbers'
+
+import { APRTooltipContent } from '../FarmingPoolAPRCell'
+import { useSharePoolContext } from '../SharePoolContext'
+import FeeTarget from './FeeTarget'
+import PositionDetail from './PostionDetail'
+import { FeeTag } from './styleds'
+
+const FlipCard = styled.div<{ flip: boolean }>`
+  border-radius: 20px;
+  padding: 16px;
+  background-image: url(${bgimg});
+  background-size: cover;
+  background-repeat: no-repeat;
+  background-color: ${({ theme }) => theme.buttonBlack};
+  position: relative;
+  transition: transform 0.6s;
+  transform-style: preserve-3d;
+
+  transform: rotateY(${({ flip }) => (flip ? '-180deg' : '0')});
+`
+
+const FlipCardFront = styled.div`
+  display: flex;
+  height: 100%;
+  flex-direction: column;
+  backface-visibility: hidden;
+  -webkit-backface-visibility: hidden;
```

Comment:
This line is not needed. SC will add the prefix automatically for us.

Reply:
updated.